src/containers/Nav/Nav.js

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment } from 'react';
+import { NavLink } from 'react-router-dom';
+import { setFilter, setMovies } from '../../actions';
+import { connect } from 'react-redux';
+import apiKey from '../../API_KEY/movie_api_key';
+import './Nav.scss'
+import Button from '../Button/Button'
+
+
+
+const Nav = (props) => {
+
+  const fetchMovies = async () => {
+    const response = await fetch(`https://api.themoviedb.org/3/movie/upcoming${apiKey}`);
+    const movies = await response.json();
+    await props.setMovies(movies);
+    return movies;
+  }
+
+  return (
+    <nav className='Nav'>
+      < button onClick={fetchMovies}> Click Me and check Redux State</button >
```

Comment:
Is this an instance where we could use a regular Link component instead of a button?